xml/admin_gui_dashboard.xml

```diff
@@ -2424,9 +2430,16 @@ Please consult the documentation on how to configure and enable the Object Gatew
     <title>Adding a New Bucket</title>
     <para>
      To add a new &ogw; bucket, click <guimenu>Create</guimenu> in the top left of
-     the table heading. Enter the bucket's name and select its owner. Confirm
-     with <guimenu>Create Bucket</guimenu>.
+     the table heading. Enter the bucket's name, select the user an owner, and
```

Comment:
```suggestion
     the table heading. Enter the bucket's name, select the user and owner, and
```

Reply:
I am guessing you meant "and" but ISTM it could have been "or"...

Reply:
oh that's an odd one. I didn't mean either. Essentially, you seelct the user *for* the ownership (of the bucket). But in the gui, you jsut select a user for the owner. See: 
![Screenshot from 2020-08-27 16-45-39](https://user-images.githubusercontent.com/1987994/91464992-235fd980-e885-11ea-9c7c-d218a447b13b.png)

Thoughts?

Reply:
How about just "select the owner"?